Block/Js.php

```diff
@@ -54,14 +54,17 @@ protected function _toHtml()
      *
      * @return string
      */
-    protected function generateJsIncludeString()
+    protected function getJsScriptsString()
     {
-        $moduleVersionComment   = '<!-- BreadCheckout Module Version: %s -->';
-        $breadJsScriptTag       = '<script src="%s" data-api-key="%s"></script>';
-        $html                   = sprintf($moduleVersionComment . $breadJsScriptTag, $this->getModuleVersion(),
-            $this->getJsLibLocation(), $this->getPublicApiKey());
-      
-        return $html;
+        $moduleVersionComment = sprintf('<!-- BreadCheckout Module Version: %s -->', $this->getModuleVersion());
+
+        $sentryScripts = sprintf('<script src="https://browser.sentry-cdn.com/5.4.3/bundle.min.js" crossorigin="anonymous"></script>'
+            . '<script src="%s"></script>',
+            $this->getViewFileUrl('Bread_BreadCheckout/js/sentry-config.js'));
```

Comment:
What is the reason for loading component this way? Also, you can load sentry from layout xml.

Reply:
oh ok sweet, yeah if that's the standard way to load it in then I'll do that